adopt-a-dog/dogs.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+  <meta charset="utf-8">
+  <title>Adopt a Dog</title>
+</head>
+<body>
+    <header>
+        <div class="title">
+          <img src="adoptadog-logo.svg" alt="ADOPTADOG" class="logo">
+          <h1>Adopt A Dog</h1>
+          <h2>Seattle's premiere dog adoption agency</h2>
+        </div>
+        <div class="checkout">
+            <h6><!--shopping cart--> $1234.56</h6>
+            <h6>Checkout</h6>
+        </div>
+          <nav>
+            <ul class="nav">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="dogs.html">Dogs</a></li>
+                <li><a href="blog.html">Blog</a></li>
+            </ul>
+          </nav>
+      </header>
+      <article>
+          <div class="grid-container">
+              <div class="grid-Murphy">
```

Comment:
What I would do is use the same class for all of the dogs, that way you can apply the same styles to each dog.